apps/coscrad-frontend/src/components/authentication-button/authentication-button.tsx

```diff
@@ -1,11 +1,16 @@
 import { useAuth0 } from '@auth0/auth0-react';
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
 import LoginButton from '../login-button/login-button';
 import LogoutButton from '../logout-button/logout-button';
 
 const AuthenticationButton = () => {
     const { isAuthenticated } = useAuth0();
 
-    return isAuthenticated ? <LogoutButton /> : <LoginButton />;
+    const { enableLoginButton } = useContext(ConfigurableContentContext);
+
+    if (enableLoginButton === true) return isAuthenticated ? <LogoutButton /> : <LoginButton />;
```

Comment:
if(enableLoginButton) is exactly the same as if(enableLoginButton === true). No need to ever compare to a boolean just write the boolean expression.